### PR TITLE
Change gradient legend max/min to relative units

### LINKED ISSUE #427: Change gradient legend max/min to relative units

### PR DESCRIPTION
**Pull request by hojelse**
Fixes #427

### CODE REVIEW

**Comment by joglr** [comment]
I think you still need to handle this separately for "Most commits"
image

**Review by joglr — DISMISSED** [review]
I think you still need to handle this separately for "Most commits"
image

**Comment by hojelse** [comment]
Lol how did this happen

File: src/components/Legend.tsx [hunk 1]
```diff
@@ -92,8 +93,8 @@ export function GradientMetricLegend({ metricType, metricCaches }: MetricLegendP
   return (
     <>
       <GradientLegendDiv>
-        <LegendLable>{minValue}</LegendLable>
-        <LegendLable>{maxValue}</LegendLable>
+        <LegendLable title={dateFormatLong(minValue)} >{dateFormatRelative(minValue)}</LegendLable>
```
[the remark this answers is not in the document]
Comment (hojelse) [reply src/components/Legend.tsx:96]: Oh I see, will fix

[the remark this answers is not in the document]
Comment (joglr) [reply src/components/Legend.tsx:96]: Cool, just rerequest review when you are done 🎉

[the remark this answers is not in the document]
Comment (joglr) [reply src/components/Legend.tsx:96]: Potential issue: This API might not be available when server-side rendering 🤷🏼‍♂️

**Review by tjomson — APPROVED** [review]
Legend line 66 and 67 could use self-closing components, and GradientMetricLegend does not need to be exported (and it should be with a lower case g)

But these are not related to your work, so you can fix it if you want, but otherwise 🚀

**Pull request merged**